test/ipc.ts

```diff
@@ -91,7 +91,7 @@ export function fakeIpc(
         ),
       )
     }),
-  }
+  } as unknown) as Partial<IpcRenderer>
```

Comment:
given that all that happens to these below is that they get cast as unknown again and then cast as `IpcMain` and `IpcRenderer`, is there a simpler way to do this? Seems excessive to cast back and forth so many times

Reply:
Yeah, I don't like any of this. It should be improved or replaced.